doc/modules/ROOT/pages/appendix/gds-session.adoc

```diff
@@ -0,0 +1,298 @@
+
+
+=== Example
+
+This example shows how to project a graph into a GDS Session.
+The example graph is heterogeneous and models users and products.
+Users can know each other, and users can buy products.
+The database connection is to a new, empty AuraDB instance.
+
+.First, we create a GDS Session and add some data to our database.
+[source,python,role=no-test]
+----
+import os # for reading environment variables
+from graphdatascience.session import SessionSizes, DbmsConnectionInfo, GdsSessions, AuraAPICredentials
+
+sessions = GdsSessions(ds_connection=AuraAPICredentials(os.environ["CLIENT_ID"], os.environ["CLIENT_SECRET"]))
+
+db_connection = DbmsConnectionInfo(os.environ["DB_URI"], os.environ["DB_USER"], os.environ["DB_PASSWORD"])
+gds = sessions.get_or_create(
+    session_name="my-new-session",
+    size=SessionSizes.by_memory().DEFAULT,
+    db_connection=db_connection,
+)
+
+gds.run_cypher(
+    """
+    CREATE
+     (u1:User {name: 'Mats'}),
+     (u2:User {name: 'Florentin'}),
+     (p1:Product {name: 'ice cream', cost: 4.2}),
+     (p2:Product {name: 'computer', cost: 13.37})
+
+    CREATE
+     (u1)-[:KNOWS {since: 2020}]->(u2),
+     (u2)-[:BOUGHT {price: 7474}]->(p1),
+     (u1)-[:BOUGHT {price: 1337}]->(p2)
+    """
+)
+----
+
+With the `gds` GDS Session active, project the graph and specify node and relationship property schemas as follows:
+
+[source,python,role=no-test]
+----
+from graphdatascience.session import GdsPropertyTypes
+
+G, result = gds.graph.project(
+    "my-graph",
+    """
+    CALL {
+        MATCH (u1:User)
+        OPTIONAL MATCH (u1)-[r:KNOWS]->(u2:User)
+        RETURN u1 AS source, r AS rel, u2 AS target, {} AS sourceNodeProperties, {} AS targetNodeProperties
+        UNION
+        MATCH (p:Product)
+        OPTIONAL MATCH (p)<-[r:BOUGHT]-(user:User)
+        RETURN user AS source, r AS rel, p AS target, {} AS sourceNodeProperties, {cost: p.cost} AS targetNodeProperties
+    }
+    RETURN gds.graph.project.remote(source, target, {
+      sourceNodeProperties: sourceNodeProperties,
+      targetNodeProperties: targetNodeProperties,
+      sourceNodeLabels: labels(source),
+      targetNodeLabels: labels(target),
+      relationshipType: type(rel),
+      relationshipProperties: properties(rel)
+    })
+    """,
+    nodePropertySchema={"cost": GdsPropertyTypes.DOUBLE},
+    relationshipPropertySchema={"since": GdsPropertyTypes.LONG, "price": GdsPropertyTypes.DOUBLE},
+)
+----
+
+
+== Running algorithms
+
+You can run algorithms on a remotely projected graph in the same way as you would on any other graph.
+For a full list of available algorithms, see the https://neo4j.com/docs/graph-data-science-client/{page-version}/api/[API reference].
+
+=== Example 
+
+Extending the previous example, we will now run the PageRank and FastRP algorithms on the projected graph.
+Then we will stream the results back joined with the `name` property fetched from the db.
```

Comment:
```suggestion
You can run algorithms on a remotely projected graph in the same way you would on any projected graph.
For instance, you can run the PageRank and FastRP algorithms on the projected graph from the previous example as follows:
```

Reply:
So you are recommending to not have extra `Example` subheadings for each section?

Reply:
yeah let's skip the additional subheadings

Reply:
Yes, just because there's not much under the heading, nor it is a separate example.